CONTRIBUTING.md

```diff
@@ -295,6 +295,74 @@ ols/app/endpoints/feedback.py                    45     24    47%   37-39, 51, 6
 ...
 ```
 
+
+
+## Benchmarks
+
+### Running benchmarks
+
+Bechmarks for selected functions and methods can be run by using following command:
+
+```
+pdm run benchmarks
+```
+
+### `pytest-benchmark` package
+
+Benchmarks are based on `pytest-benchmark` package. To create new benchmark, the regular test function needs
+to be created and `benchmark` parameter (fixture) should be passed to it. Then the benchmark can be initialized
+by calling this fixture, passing tested function/method to it, together with required parameters.
+
+### Basic usage of `benchmark` fixture
+
+```python
+def test_format_attachment_plain_text_format(benchmark):
```

Comment:
For the sake of completeness?

`from ols.src.query_helpers.attachment_appender import format_attachment`